src/webview/wkwebview/mod.rs

```diff
@@ -663,38 +663,13 @@ r#"Object.defineProperty(window, 'ipc', {
       // Inject the web view into the window as main content
       #[cfg(target_os = "macos")]
       {
-        let parent_view_cls = match ClassDecl::new("WryWebViewParent", class!(NSView)) {
-          Some(mut decl) => {
-            decl.add_method(
-              sel!(keyDown:),
-              key_down as extern "C" fn(&mut Object, Sel, id),
-            );
-
-            extern "C" fn key_down(_this: &mut Object, _sel: Sel, _event: id) {
-              // do nothing; this prevents macOS from playing an unsupported key feedback sound
-            }
```

Comment:
does this bug happen with winit and this PR?

Reply:
No, TaoView/WinitView also delcared this method already.